src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -299,5 +311,21 @@ private JObject SignAndRelay(Transaction tx)
                 return context.ToJson();
             }
         }
+
+        internal static UInt160 StringToScriptHash(string input)
```

Comment:
Maybe it is better to create a function just for AssetID and other function for scripthash.

Reply:
To AssetID can use UInt160.Parse() directly, now this just for scripthash.